src/components/carsoulSection/StylecarsoulSection/myStyle.scss

```diff
@@ -0,0 +1,86 @@
+@import '../../../assets/main.css';
+
+$gray-color: #979797;
+$pink-color: #eec197;
+$light-blue: #8bb4ea;
+$dark-blue: #7aa2d6;
+$light-gray: #f6f6f6;
+$light-blue2: #f9f9f9;
+$font-roboto: 'Roboto';
+
+.mycolorPink {
+  background: $pink-color;
+}
+
+%fontType {
+  font: 'Saira';
+}
+
+.fontHeader1 {
+  @extend %fontType;
+  color: $dark-blue;
+}
+
+.fontHeader2 {
+  @extend %fontType;
+  color: $gray-color;
+  font-size: 1.1rem;
+}
+
+.parg1 {
+  color: $gray-color;
+  font: $font-roboto;
+  font-size: 0.9rem;
+}
+
+.fontPrice {
+  font: $font-roboto;
+  color: #333333;
+  font-size: 1.5rem;
+}
+
+.buttomText {
+  font: $font-roboto;
+  font-size: 1rem;
+}
+
+.highlight {
+  font: $font-roboto;
+  width: 15rem;
+  height: 2.3rem;
+}
+
+.bgGray {
+  background: $light-gray;
+}
+
+.bgLightblue {
+  background: $light-blue2;
+}
+
+.lightblue {
+  background: $light-blue;
+}
+
+.imgGrideCarsouel {
+  display: block;
+  width: 100%;
+  margin: 0 auto 0 auto;
+  height: 9rem;
+}
+
+.imageCardCarsouel {
+  display: block;
+  width: 100%;
+  height: 19rem;
+}
+
+@media (max-width: 640px) {
```

Comment:
don't use media query like this and instead use the scss method [Check out the first part of this](http://thesassway.com/intermediate/responsive-web-design-in-sass-using-media-queries-in-sass-32)

or if you want to go more advance in it so you should [checkout this](https://medium.com/codeartisan/breakpoints-and-media-queries-in-scss-46e8f551e2f2)